.github/workflows/ci.yml

```
@@ -29,16 +29,29 @@ jobs:
         pip-version:
           - "latest"
           - "20.0"
+        include:
+          - os: Ubuntu
+            python-version: 3.9-dev
+            pip-version: latest
+
     env:
       PY_COLORS: 1
       TOXENV: pip${{ matrix.pip-version }}-coverage
       TOX_PARALLEL_NO_SPINNER: 1
     steps:
       - uses: actions/checkout@master
-      - name: Set up Python ${{ matrix.python-version }}
+      - name: Set up Python ${{ matrix.python-version }} from GitHub
+        if: "!endsWith(matrix.python-version, '-dev')"
```

my preference is to avoid extra quotes but it's up to you
```suggestion
        if: >-
          !endsWith(matrix.python-version, '-dev')
```